.travis.yml

```diff
@@ -48,6 +47,8 @@ jobs:
         - make install-dev
       before_script:
         - oc login $OPENSHIFT_URL --token=$OPENSHIFT_TOKEN
+        - export CLOUDIGRADE_USER=$(python -c "from uuid import uuid4; print(str(uuid4()))")
```

Comment:
Do you know the `uuidgen` command?

Reply:
Yeah I use that locally but then it broke on travis in a kind of silent way, because uuidgen was not present. I was not sure about what package to install/with the move to gitlab how that would change, but I know we have python :stuck_out_tongue_winking_eye: